utils/mock-data-loader/api.js

```diff
@@ -124,12 +125,13 @@ const createEligibilityCriteria = async (eligibilityCriteria, token) => {
   return response.data;
 };
 
-const createUser = async (user) => {
+const createUser = async (user, token) => {
   const response = await axios({
     method: 'post',
     headers: {
       'Content-Type': 'application/json',
       Accepts: 'application/json',
+      Authorization: token || '',
```

Comment:
Do we need to an empty string if `token` is falsey? Also applies to all of the below aswell.

Reply:
I'm not sure, I was following the same pattern as in the rest of this file. But I think since this is only called from within `mock-data-loader` we wouldn't expect the token to be falsey so maybe shouldn't account for it.

Should I remove the `|| ''` from all of these endpoints?

Reply:
Please remove.